source/release-notes/backend.txt

```diff
@@ -13,6 +13,33 @@ Atlas App Services Changelog
 2024 Releases
 -------------
 
+.. _backend_20240417:
+
+17 April 2024 Release
+~~~~~~~~~~~~~~~~~~~~~
+
+- Fixed Google Cloud Provider ``us-central1`` label to read "Iowa" instead of "Ohio" in App Services Deployment
+  Region settings.
+
+- Improved user caching for authentication provider user metadata. For more information, refer to
+  notes in :ref:`Authentication Provider Metadata <auth-provider-metadata>`.
+
+- There are now three types of private endpoints: general, sync, and legacy. Legacy endpoints must migrate
+  to either general or sync. For more information, refer to :ref:`Use a VPC Private Endpoint <private_endpoints>`.
+
+- For the CLI, added a prompt to pick from available app users when using the ``--user`` flag. 
+
+- App Services now uses the ``services.cloud.mongodb.com`` domain rather than the ``realm.mongodb.com`` domain.
```

Comment:
```suggestion
- App Services now uses the ``services.cloud.mongodb.com`` domain rather than the ``realm.mongodb.com`` domain. For more information, refer to :ref:`realm-to-cloud-domain-migration`.
```
should this be at the top of the RN list? seems important